src/plugins/intel_cpu/src/transformations/snippets/normalize_convert_pre_lower.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "transformations/snippets/normalize_convert_pre_lower.hpp"
+
+#include <openvino/op/convert.hpp>
+#include <openvino/opsets/opset1.hpp>
+#include <openvino/pass/manager.hpp>
+#include <openvino/pass/pattern/matcher.hpp>
+#include <openvino/pass/pattern/op/wrap_type.hpp>
+
+#include "snippets/op/convert_truncation.hpp"
+#include "snippets/op/subgraph.hpp"
+
+namespace ov::intel_cpu::pass {
+
+namespace {
+class ReplaceConvertInBody : public ov::pass::MatcherPass {
+public:
+    OPENVINO_MATCHER_PASS_RTTI("CPU::ReplaceConvertInBody");
+    ReplaceConvertInBody() {
+        auto m_v0 = ov::pass::pattern::wrap_type<ov::op::v0::Convert>();
+        auto m_v1 = ov::pass::pattern::wrap_type<ov::opset1::Convert>();
```

Comment:
Could you please try this option?
```suggestion
        auto convert_m = ov::pass::pattern::wrap_type<ov::op::v0::Convert, ov::opset1::Convert>();
```
If it works for android, I'd prefer leaving it since it is shorter